### ISSUE TITLE
schema.schema.gql.dart not generated - "possibleTypesMap" missing

### ISSUE THREAD

**Issue by simplenotezy** [gql-dart/ferry#594]
I am following the example repos, but with my own GraphQL endpoint. I notice that when I run: `get-graphql-schema http://localhost:8787 > lib/common/domain/schema.graphql`

And then then buildrunner: `dart run build_runner build  --delete-conflicting-outputs`, it generates all files, except the `schema.schema.gql.dart` file:

<img width="507" alt="image" src="https://github.com/gql-dart/ferry/assets/3786627/3b187445-3ca7-48b2-8bb5-3102c1eba480">

(it even creates the `schema.schema.gql.g.dart` file?)

What am I missing here?

**Comment by knaeckeKami** [comment]
There's an old bug that occurs when a schema has no inputs, no enums and no scalars which causes this - does your schema maybe look like this?

**Comment by simplenotezy** [comment]
Yes, that was the case @knaeckeKami thanks! It started working suddenly once I started evolving the defined schema. Thanks

**Comment by adamhb123** [comment]
I'm having this issue as well now - it only happens after adding a JsonSerializer for the JSON scalar type. Currently, I am just setting possibleTypesMap to an empty map ( {} ). Is this an appropriate solution? I'm not quite sure what it is used for but my queries seem to work just fine without it.
